test/functional/inspec_exec_json_test.rb

```diff
@@ -158,9 +145,15 @@
       })
 
       _(groups.sort_by { |x| x["id"] }).must_equal([
-        { "id" => "controls/example-tmp.rb", "title" => "/tmp profile", "controls" => ["tmp-1.0", key] },
-        { "id" => "controls/example.rb", "title" => "Example Config Checks", "controls" => ["example-1.0"] },
-        { "id" => "controls/meta.rb", "title" => "SSH Server Configuration", "controls" => ["ssh-1"] },
+        { "id" => "controls/example-tmp.rb",
```

Comment:
Should we s/tmp/slash/ or s/tmp/root/ since we're no longer dealing with tmp? There are a bunch of 'tmp' remnants in here.

Reply:
sames